DatWeatherDoe/UI/Decorator/Text/RiseSet/RiseSetTextBuilder.swift

```diff
@@ -0,0 +1,68 @@
+//
+//  RiseSetTextBuilder.swift
+//  DatWeatherDoe
+//
+//  Created by Markus Markus on 7/26/22.
+//  Copyright © 2022 Markus Mayer.
+//
+
+import Foundation
+
+final class RiseSetTextBuilder {
+
+    private let initial: String
+    private let sunset: TimeInterval
+    private let sunrise: TimeInterval
+
+    private let upArrowStr = "⬆"
+    private let downArrowStr = "⬇"
+
+    init(
```

Comment:
Do we absolutely need defaults in the initializer here? This initializer is also redundant and we get this for free

Reply:
Not totally certain what you mean. The only change I can make that won't lead to a compiler error is removing the initial values for sunrise, sunset.

Like so:

init(initial: String = "", sunset: TimeInterval, sunrise: TimeInterval)
{ ... }

Reply:
This is as simple I was able to make it. If I remove anything else from the initializer (or the initializer itself), Xcode will complain.

```
    private let sunset: TimeInterval
    private let sunrise: TimeInterval
...
    init(
        sunset: TimeInterval,
        sunrise: TimeInterval
    ) {
        self.sunset = sunset
        self.sunrise = sunrise
    }

```